From 7e3001f3546e20f22aa7d0dc887a83ded1d611b7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20Sawicz?= <michal@sawicz.net>
Date: Fri, 1 Aug 2025 12:46:00 +0000
Subject: [PATCH] DisplayConfigurationStorage: drop use of descriptors

These are a repeat of the properties already used above,
and we'd like to remove them in Mir.
---
 src/DisplayConfigurationStorage.cpp | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/DisplayConfigurationStorage.cpp b/src/DisplayConfigurationStorage.cpp
index 4d8d4aede..5a785b531 100644
--- a/src/DisplayConfigurationStorage.cpp
+++ b/src/DisplayConfigurationStorage.cpp
@@ -33,9 +33,6 @@ inline QString stringFromEdid(const miroil::Edid& edid)
     str += QString::fromStdString(edid.vendor);
     str += QString("%1%2").arg(edid.product_code).arg(edid.serial_number);
 
-    for (int i = 0; i < 4; i++) {
-        str += QString::fromStdString(edid.descriptors[i].string_value());
-    }
     return str;
 }
 
-- 
2.51.0

